### ISSUE TITLE
HTMLtoJSX does not convert curly brackets

### ISSUE THREAD

**Issue by optimalisatie** [reactjs/react-magic#42]
> _Issue copied from [React:Issues](https://github.com/facebook/react/issues/5910)._

I am experiencing a problem with the HTMLtoJSX converter (https://www.npmjs.com/package/htmltojsx), the same converter as officially displayed on https://facebook.github.io/react/html-jsx.html.

The HTML-code `<div>{</div>` is compiled to the following JSX-code:

```jsx
<div>{</div>
```
https://facebook.github.io/react/html-jsx.html

The JSX code fails with the following error:

```javascript
Uncaught SyntaxError: embedded: Unexpected token (7:18)
   5 | var Hello = React.createClass({
   6 |   render: function() {
>  7 |     return <div>{</div>;
     |                   ^
   8 |   }
   9 | });
```
Test: https://jsfiddle.net/69z2wepo/28714/

Does anyone know a solution for the problem?

---
> Reply from @Daniel15 
> Good catch, thanks for reporting this! As @zpao mentioned, please file this in the react-magic repo so I remember to fix it :)

**Comment by solugebefola** [comment]
I have put in a pull request with my fix.

**Comment by Daniel15** [comment]
Should be fixed by #55
